src/rgw/rgw_op.cc

```diff
@@ -3914,7 +3914,7 @@ void RGWDeleteLC::execute()
   store->get_bucket_instance_obj(s->bucket, obj);
   store->set_atomic(s->obj_ctx, obj);
   ret = get_system_obj_attrs(store, s, obj, orig_attrs, NULL, &s->bucket_info.objv_tracker);
-  if (op_ret < 0)
```

Comment:
I don't know why  but I think we can remove it , and I found the class ``RGWGetLC  `` and `` RGWPutLC``  has the field ret too,   may be we should open other PR to fix them, what would you think?